policy-service/README.md

```diff
@@ -20,8 +20,22 @@ limitations under the License.
 
 # Policy Service
 
-The responsibility of the Policy Service is to provide the set of rules (filters or transformations) that need to be
-applied to each resource that has been requested, based the User and Context.
+The responsibility of the Policy Service is to provide the "set of rules" (filters or transformations) that need to be applied to each resource that has been requested, based the User and Context.
+The "set of rules" are split into 2 groups:
+1. Resource Level Rules (If no resource level rules are set on a resource then access will not be granted to any users)
+1. Record Level Rules
```

Comment:
Same applies here, no record-level rules means access denied

Reply:
I've added the same information as above for the record level rules